sdk/monitor/Azure.Monitor.OpenTelemetry/tests/Examples.AspNetCore/Examples.AspNetCore.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Azure.Monitor.OpenTelemetry.csproj" />
```

Comment:
```suggestion
    <ProjectReference Include="..\..\src\Azure.Monitor.OpenTelemetry.AspNetCore.csproj" />
```

Reply:
Waiting for product team decision on Nuget package name.. They wanted to use `Azure.Monitor.OpenTelemetry` for ASP.NET Core and `Azure.Monitor.OpenTelemetry.Web` for classic ASP.NET. Will work on the rename once we have the decided on the names.